src/api/api.dto.ts

```diff
@@ -54,7 +54,7 @@ export function networkToChain(network: Network): Chain {
       return ChainEnum.POLYGON
     case Network.POLYGON_MUMBAI:
       return ChainEnum.POLYGON_MUMBAI
-    case Network.CARDANO:
+    case Network.CARDANO_ROSETTA:
       return ChainEnum.POLYGON_MUMBAI
```

Comment:
Maybe you are returning incorrect chain?

Reply:
Looking at the changes, I don't think Cardano should be returned from this function at all